packages/design-system-scripts/gulp/docs/runWebpackStatically.js

```diff
@@ -0,0 +1,46 @@
+/**
```

Comment:
My notes on this:

[Referring to the old scripts,] I just realized how webpack works differently between the static build and the _start_ command. When you run `yarn start`, the underlying gulp watch task starts the `server` task, and that runs its own webpack instance that is separate from the one in the `webpack` gulp task in `gulp/webpack.js`. This was confusing! So I'm going to try to make that less confusing by naming better. I'm going to call them `docs/runWebpackStatically.js` and `docs/runWebpackServer.js`.

Reply:
thank you thats a helpful explanation!